templates/partial/_footer.html

```diff
@@ -110,11 +103,9 @@ <h2 class="p-heading--5">Blog</h2>
 
       <div class="row u-sv3">
         <div class="col-3">
-          <a class="p-heading--5 p-link--soft" href="https://charmhub.io "><small>All operators</small></a>
-        </div>
-        <div class="col-3">
-          <a class="p-heading--5 p-link--soft" href="#"><small>More comparisons</small></a>
+          <a class="p-heading--5 p-link--soft" href="/"><small>All operators</small></a>
         </div>
+        <div class="col-3"></div>
```

Comment:
Should really use empty columns but ok in a rush.

Reply:
I can update. I wasn't sure what do here to be honest :sweat: